hooks/terraform-validate.sh

```diff
@@ -7,9 +7,18 @@ set -e
 # workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
 export PATH=$PATH:/usr/local/bin
 
+# Disable output not usually helpful when running in automation (such as guidance to run plan after init)
+export TF_IS_AUTOMATION=1
```

Comment:
```suggestion
export TF_IN_AUTOMATION =1
```

Reply:
This results in an error, as bash doesn't allow spaces around the operand for assignments:
```
$ export TF_IN_AUTOMATION =1
-bash: export: `=1': not a valid identifier
```

Reply:
Sorry, the space was a copy/paste error. The relevant change is the variable name: `IN` versus `IS`.

Reply:
Space fixed

Reply:
Oh, nvm…I just noticed I mistyped the variable on my final commit 🤦🏻 I'll fix this…